BREAKING.md

```diff
@@ -289,8 +289,12 @@ Any references to the virtual scroll types from `@ionic/core` have been removed.
 
 <h4 id="version-7x-angular">Angular</h4>
 
+- Angular v14 is now required to use `@ionic/angular` and `@ionic/angular-server`. Upgrade your project to Angular v14 by following the [Angular v14 update guide](https://update.angular.io/?l=3&v=13.0-14.0).
```

Comment:
We've been updating required version here too: https://github.com/ionic-team/ionic-framework/blob/feature-7.0/BREAKING.md#browser-and-platform-support